homeassistant/components/renault/sensor.py

```diff
@@ -47,19 +45,19 @@ async def async_setup_entry(
 ) -> None:
     """Set up the Renault entities from config entry."""
     proxy: RenaultHub = hass.data[DOMAIN][config_entry.unique_id]
-    entities = await get_entities(proxy)
+    entities = get_entities(proxy)
```

Comment:
Is it ok to keep `async def async_setup_entry` as async, even without `await`?

Reply:
Yeah, the `async_setup_entry` functions are part of the standardized API in Home Assistant for integrations that use config entries. So those functions always need to be defined according to the template defined by the API.